extensions/amp-story-panning-media/0.1/amp-story-panning-media.css

```diff
@@ -18,6 +18,8 @@ amp-story-panning-media {
 
 amp-story-panning-media amp-img {
   backface-visibility: hidden !important;
+  /* Width explicitly set for mobile chrome, desktop safari & desktop firefox. */
+  width: 100% !important;
```

Comment:
Just a question/suggestion, you have more context:
Should we force the height too? My worry is publishers trying to set a size for `amp-img` within the `amp-story-panning-media`. This could be done for no reason at all (I don't think it would work with this component?), or just a general `amp-img` CSS rule publishers write that would apply here too

Reply:
I think that's a good call. Added.